docs/user-guide/install/pe/docker.md

```diff
@@ -300,25 +290,39 @@ docker-compose up mytbpe
 ```
 {: .copy-code}
 
-### Upgrading from 3.1.0PE to latest version
+### Upgrade starting from 3.1.0PE
 
-Open docker-compose.yml and change version from **3.1.0PE** to **{{ site.release.pe_full_ver }}**. 
-Then call the following commands:
+Please refer to the [troubleshooting](/docs/user-guide/install/pe/docker/#troubleshooting) section in case you are upgrading from 3.0.0 or 3.0.1.
 
-```
-sudo sh -c "echo '3.1.0PE' > ~/.mytbpe-data/.upgradeversion"
-docker-compose run mytbpe upgrade-tb.sh
-```
-{: .copy-code}
+Below is example on how to upgrade from 3.1.0 to 3.1.1
 
-Start ThingsBoard:
+1. Create a dump of your database:
 
-```
-docker-compose up mytbpe
-```
-{: .copy-code}
+    ```text
+    docker-compose exec postgres sh -c "pg_dump -U postgres thingsboard > /var/lib/postgresql/data/thingsboard_dump"
```

Comment:
Please add a command how to restore from the dump if something goes wrong. Also, how to copy the file to your local file system, etc..